azurerm/internal/services/appconfiguration/app_configuration_resource.go

```diff
@@ -50,6 +50,31 @@ func resourceArmAppConfiguration() *schema.Resource {
 
 			"location": azure.SchemaLocation(),
 
+			"identity": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"type": {
+							Type:     schema.TypeString,
+							Required: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"SystemAssigned",
```

Comment:
Could we use the predefined values in the SDK instead here?
```suggestion
								string(appconfiguration.SystemAssigned),
```